### PR TITLE
Modules README CN to EN

### PR DESCRIPTION
**Pull request by jm12138**
* Modules README CN to EN

### CODE REVIEW

File: docs/docs_en/get_start/windows_quickstart.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  - Windows Start Menu -> Anaconda3 -> Anaconda Prompt
+  - <img src="../../imgs/Install_Related/windows/anaconda_prompt.png" alt="anaconda download" width="300" align="center"/>
+
+
+- Create a new conda environment
+
+  - ```shell
+    # On the command line, enter the following command to create a file named paddle_env Env environment
+    # This is for accelerated download, use Tsinghua Source
+    conda create --name paddle_env python=3.8 --channel https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/  # 这是一行命令
```
Comment (rainyfly) [inline docs/docs_en/get_start/windows_quickstart.md]: # a shell command

File: docs/docs_en/get_start/mac_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+    ```
+
+- After installing Paddle, continue to install the PaddleHub in the paddle_env environment:
+
+  - ```shell
+    # Enter the following command on the command line
+    pip install paddlehub -i https://mirror.baidu.com/pypi/simple
+    ```
+
+  - Introduction document of paddlehub: https://github.com/PaddlePaddle/PaddleHub/blob/release/v2.1/README_ch.md
```
Comment (rainyfly) [inline docs/docs_en/get_start/mac_quickstart.md]: 替换到最新的英文readme

File: docs/docs_en/get_start/windows_quickstart.md (hunk 1 above)
Comment (jm12138) [reply docs/docs_en/get_start/windows_quickstart.md]: done

File: docs/docs_en/get_start/mac_quickstart.md (hunk 2 above)
Comment (jm12138) [reply docs/docs_en/get_start/mac_quickstart.md]: done

**Review by rainyfly — APPROVED** [review]

**Pull request merged**